README.md

```diff
@@ -11,20 +11,27 @@ articles, books, ebooks, slideshows, web pages, man pages and blogs.
 AsciiDoc files can be translated to many formats including HTML, PDF,
 EPUB, man page.
 
-AsciiDoc is highly configurable: both the AsciiDoc source file syntax
+AsciiDoc.py is a legacy processor for this syntax, handling an older
+specification of AsciiDoc. As such, this will not properly handle the
```

Comment:
AsciiDoc never had a specification, so I don't think specification is the right word here. I would prefer "rendition" or, if you prefer, "syntax".

Reply:
Rendition is a good word and avoids too much repition of the word "syntax".

Changed

Reply:
Thanks!